src/main/java/net/dv8tion/jda/api/entities/sticker/StandardSticker.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2015 Austin Keener, Michael Ritter, Florian Spieß, and the JDA contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.api.entities.sticker;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Standard stickers used for nitro and wave buttons on welcome messages.
+ *
+ * <p>This includes stickers from {@link StickerPack StickerPacks}, such as wumpus or doggos.
+ */
+public interface StandardSticker extends RichSticker
+{
+    /**
+     * The ID of the pack the sticker is from.
+     *
+     * @return the ID of the pack the sticker is from
+     */
+    long getPackIdLong();
+
+    /**
+     * The ID of the pack the sticker is from.
+     *
+     * @return the ID of the pack the sticker is from
+     */
+    @Nonnull
+    default String getPackId()
+    {
+        return Long.toUnsignedString(getPackIdLong());
+    }
+
+    /**
+     * The sticker's sort order within its pack
+     *
+     * @return The sort order value
+     */
+    int getSortValue();
```

Comment:
I have no idea, the docs don't explain this at all. I'm guessing its just the index in the pack

Reply:
Gonna double check with the DDevs. I expect the same.
I don't think it is overly impactful to this PR though tbh.

Reply:
![image](https://user-images.githubusercontent.com/1479909/168402584-2802a7b2-0f30-4746-9d30-21c2ca9a7dad.png)
So not actually an index, so we should definitely leave it named as `sortValue` instead of trying to convey it as an index 👍